src/extensions/replay/sessionrecording.ts

```diff
@@ -343,13 +361,37 @@ export class SessionRecording {
             return 'buffering'
         }
 
+        if (this.urlTriggerStatus === 'pending') {
+            return 'buffering'
+        }
+
         if (isBoolean(this.isSampled)) {
             return this.isSampled ? 'sampled' : 'disabled'
         } else {
             return 'active'
         }
     }
 
+    private get urlTriggerStatus(): 'activated' | 'pending' | 'disabled' {
+        if (this.receivedDecide && this._urlTriggers.length === 0) {
+            return 'disabled'
```

Comment:
i guess disabled means url triggering is disabled and not recording is disabled... 

not blocking if you disagree - I'd be tempted to call it something like 'no-url-triggering'  to avoid (me) people getting confused in future but very very picky :)